sdk/monitorexporter/azure-analytics-monitorexporter/src/main/java/com/azure/analytics/monitorexporter/models/TrackResponse.java

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
```

Comment:
Is this a generated model?
If so, we should add this to `custom-types` on the swagger readme. 
Same for other models.